demosplan/DemosPlanCoreBundle/Logic/FileResponseGenerator/DocxResponseGenerator.php

```diff
@@ -11,13 +11,15 @@
 namespace demosplan\DemosPlanCoreBundle\Logic\FileResponseGenerator;
 
 use demosplan\DemosPlanCoreBundle\Exception\DemosException;
+use demosplan\DemosPlanCoreBundle\Logic\Procedure\NameGenerator;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpFoundation\StreamedResponse;
 
 class DocxResponseGenerator extends FileResponseGeneratorAbstract
 {
-    public function __construct(array $supportedTypes)
+    public function __construct(array $supportedTypes, NameGenerator $nameGenerator)
     {
+        parent::__construct($nameGenerator);
```

Comment:
why do you need the parent call?

Reply:
We have two __Construct and should call both. I think, we should either use parent or initialize parameter.

Reply:
My reference was too old, it is correct here